docs/References/elementTemplateSupport.md

```diff
@@ -0,0 +1,59 @@
+---
+sidebar_position: 2
+---
+import styles from './styling/elementTemplateSupport.css';
+import fileAssociations from './img/fileAssociations.png';
+import singleSchema from './img/singleSchema.png';
+import schemaEntry from './img/schemaEntry.png';
+import jsonSettings from './img/jsonSettings.png';
+
+# Template-Support
+Für einfacheres Entwickeln von Element-Templates wünschen sich Modellierer und Entwickler in der Regel zwei Dinge:
+
+1. Autocompletion
```

Comment:
Autocompletion ist auf deutsch automatische Vervollständigung. Ich würde versuchen nicht zu viele Anglizismen zu verwenden, da sonst schnell die Frage aufkommt *Warum hat man das nicht gleich in English geschrieben*